mitmproxy/addons/core.py

```diff
@@ -103,15 +103,15 @@ def resume(self, flows: typing.Sequence[flow.Flow]) -> None:
 
     # FIXME: this will become view.mark later
     @command.command("flow.mark")
-    def mark(self, flows: typing.Sequence[flow.Flow], boolean: bool) -> None:
+    @command.argument("marker", type=mitmproxy.types.Marker)
+    def mark(self, flows: typing.Sequence[flow.Flow], marker: typing.Union[bool, str]) -> None:
```

Comment:
Is there a reason why you renamed the command? I don't particularly mind the change, but we should also strive to not break compatibility too badly.

Reply:
That is a goof! Good catch. I'll fix that.